patches/chromium/add_trustedauthclient_to_urlloaderfactory.patch

```diff
@@ -39,17 +39,25 @@ index 6b14d8354375377526e141ee499a7583be3f22b0..eeb9e19c0ecdf4631e596e7c0927693f
  interface CertVerifierClient {
    Verify(
      int32 default_error,
-@@ -559,6 +578,8 @@ struct URLLoaderFactoryParams {
-   // impact because of the extra process hops, so use should be minimized.
-   pending_remote<TrustedURLLoaderHeaderClient>? header_client;
+@@ -597,6 +616,16 @@ struct URLLoaderFactoryParams {
+   // interface. This still respects the per-context block lists.
+   CorsOriginAccessPatterns? factory_bound_access_patterns;
  
 +  pending_remote<TrustedURLLoaderAuthClient>? auth_client;
 +
-   // If non-empty array is given, |factory_bound_allow_patterns| is used for
-   // CORS checks in addition to the per-context allow patterns that is managed
-   // via NetworkContext interface. This still respects the per-context block
++  // If non-empty array is given, |factory_bound_allow_patterns| is used for
++  // CORS checks in addition to the per-context allow patterns that is managed
++  // via NetworkContext interface. This still respects the per-context block
++  // lists. To use this feature, caller should set |request_initiator_site_lock|
++  // in order to bind the allow patterns to the factory specific initiator
++  // origin.
++  array<CorsOriginPattern> factory_bound_allow_patterns;
```

Comment:
This looks like a mis-resolved conflict; I don't think we intend to add this value?

Reply:
Will check, I didn't touch this one.